### ISSUE TITLE
Process name when attaching debugger

### ISSUE THREAD

**Issue by alanbirtles** [microsoft/vscode-cpptools#909]
I would be nice if there was an option to preselect a process name for "${command:pickProcess}", I'm always attaching to the same process and it would be good to not have to type the process name every time

**Comment by pieandcakes** [comment]
@alanbirtles This is something we can look at but we're expecting processIDs when attaching and not process names.

**Comment by alanbirtles** [comment]
no need to change that, I was just asking for a change to the process picker to either have a predefined process name filter or to remember the last process name and use that as the filter

**Comment by CelticMinstrel** [comment]
The ability to set a predefined filter would be very helpful to me too. Maybe the `extension.pickNativeProcess` command can be set up to accept arguments so we can use the `"inputs"` functionality for this?

**Comment by Spongman** [comment]
> we're expecting processIDs when attaching

are you expecting the user tp type in process IDs into the UI?

is that an expectation of a good user experience?

**Comment by adriancostin6** [comment]
See my pitch for this here https://github.com/microsoft/vscode-cpptools/issues/6134#issuecomment-1748519614. I just saw that the other issue was closed as a duplicate of this one.

**Comment by avierr** [comment]
having to search for the process name every time is really annoying.

**Comment by Spongman** [comment]
yeah, you would have thought that if there's only one debuggable process running whose name matches the binary you're debugging, then the default behavior should be to just debug that one, surely?
